### DISCUSSION TITLE
Git clone task config sections

### DISCUSSION THREAD

**Discussion by joeshannon** [eclipse-oomph/oomph#67] [Q&A] [answered]
Hi,
We have configuration in our git clone tasks to add additional ref spec to a particular origin (for Gerrit purposes), it looks like:
![image](https://github.com/eclipse-oomph/oomph/assets/35498448/32b23012-2f24-428d-8be7-cd18e5a76581)

A couple of weeks ago developers started noticing that the "Fetch from Gerrit..." option was not available anymore for newly provisioned workspaces. In the new git clones, only the first fetch refspec is added to the git repository config file instead of both of them. To be explicit, the relevant section in the config file looks like:
```
[remote "origin"]
	url = ssh://gerrit:29418/repo
	fetch = +refs/heads/*:refs/remotes/origin/*
	pushurl = ssh://gerrit:29418/repo
	push = HEAD:refs/for/main
```
Instead of:
```
[remote "origin"]
	url = ssh://gerrit:29418/repo
	fetch = +refs/heads/*:refs/remotes/origin/*
	fetch = refs/notes/*:refs/notes/*
	pushurl = ssh://gerrit:29418/repo
	push = HEAD:refs/for/main
```

I noted that there have been some changes to the git Oomph plugins recently so was wondering how we might achieve the previous behaviour?

**Comment by merks** [comment, marked as answer]
It’s possible that I broke something while rewriting the complex logic. Sorry.   Please open an issue. I can’t look until at least Friday but will aim to have it fixed before m3 next week.

**Reply by joeshannon** [reply]
Thanks Ed - have created #68
